Documentation/developers/profiler.rst

```diff
@@ -0,0 +1,122 @@
+Profiler Framework
+^^^^^^^^^^^^^^^^^^
+
+Simple, easy to use and fast C++ header class framework for profiling and benchmark.
+
+Author: Simone Magnani - s41m0n
+
+Introduction
+************
+
+This framework is born to accomplish an efficient measurement of user's programs. More in detail, there already are hundreds of tools (especially in Linux) to perform benchmarks and profiling, but I noticed that they sometimes are too hard to learn and they may not meet users' needs.
```

Comment:
==> This framework provides an efficient and simple benchmarking primitive for user programs. Particularly, this can be used to profile the  control plane portion of Polycube.
Although there are several tools (particularly in Linux) that target benchmarking and profiling, ...